google/cloud/spanner/internal/partial_result_set_reader.cc

```diff
@@ -83,10 +83,15 @@ StatusOr<optional<Value>> PartialResultSetReader::NextValue() {
 
 PartialResultSetReader::~PartialResultSetReader() {
   if (!finished_) {
+    // If there is actual data in the streaming RPC Finish() can deadlock, so
```

Comment:
Should the edits in this file be in a separate PR?

Reply:
Maybe... It hangs without them, so we need them before or with this PR.

Reply:
OK.  I'm fine with them here, but let's tag @mr-salty so he knows.